328-340.ipynb

```diff
@@ -0,0 +1,356 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "02a6cbe8",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[3, 2, 12, 4, 15, 9, 5]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def prob_328(input_list):\n",
+    "    return [num for num in input_list if input_list.count(num)==1]\n",
+    "input_list = [3, 2, 1, 10, 1, 12, 4, 10, 1, 15, 1, 9, 1, 5]\n",
+    "prob_328(input_list)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "df1c22f8",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "5"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def prob_329(input_list, index):\n",
+    "    if len(input_list) >= index:\n",
+    "        sum_num = 0\n",
+    "        for i in range(index - 1):  \n",
+    "            sum_num += input_list[i]\n",
+    "        return sum_num\n",
+    "    return sum([num for num in input_list if len(input_list) >= index])\n",
```

Comment:
Can you please explain why when len(input_list) <  index we are returning return sum([num for num in input_list if len(input_list) >= index]) ?

Reply:
Thank you for the review!
At first I tried to use list comprehension, then I decided to separate. The last return should be deleted, I forgot to delete it.